Update docs to Ika (#410)
dwallet-labs/dwallet-network: docs/docs/overview.md

# Overview

## Welcome to Ika

> **Ika** is the first sub-second MPC network, delivering unmatched scalability with the ability to scale to 10,000 signatures per second (tps) and support hundreds or even thousands of signer nodes with zero-trust, noncollusive security. Utilizing [dWallets](core-concepts/dwallets) - programmable, decentralized signing mechanisms - as building blocks, builders can control native assets in other chains directly from their smart contract in a secure way.

Reviewer: ```suggestion
**Ika** is the first sub-second MPC network, delivering unmatched scalability with the ability to scale to 10,000
signatures per second (tps) and support hundreds or even thousands of signer nodes with zero-trust, noncollusive
security.
Utilizing [dWallets](core-concepts/dwallets) — programmable, decentralized signing mechanisms — as building
blocks, builders can control native assets in other chains directly from their smart contract securely.
```

## Unique Value of Ika

> Ika was created by some of the world’s leading cryptography experts and is redefining the state of the art for MPC networks:

Reviewer: ```suggestion
Ika was created by some of the world’s leading cryptography experts and is redefining the state of the art for MPC
networks:
```

> 1. **High Throughput**: Ika can scale to process thousands of signatures per second, making it the first scalable decentralized MPC infrastructure.
> 2. **Low Latency**: While legacy MPC networks lag with high latency, Ika can operate at sub-second speeds, enabling real-time applications across chains.
> 3. **Decentralization**: Ika transcends the node limit of MPC networks, and can scale to hundreds of nodes generating signatures together.
> 4. **Zero-Trust Security**: Ika allows builders to cryptographically require the user in the signature generation process, eliminating trust and ensuring user asset security.

Reviewer: ```suggestion
1. **High Throughput**: Ika can scale to process thousands of signatures per second, making it the first scalable
   decentralized MPC infrastructure.
2. **Low Latency**: While legacy MPC networks lag with high latency, Ika can operate at sub-second speeds, enabling
   real-time applications across chains.
3. **Decentralization**: Ika transcends the node limit of MPC networks, and can scale to hundreds of nodes generating
   signatures together.
4. **Zero-Trust Security**: Ika allows builders to cryptographically require the user in the signature generation
   process, eliminating trust and ensuring user asset security.
```

> By doing that, Ika provides builders with the flexibility to implement any logic (custody, DeFi, gaming etc.) across any blockchain, including Bitcoin. Builders can now create dapps that have:

Reviewer: ```suggestion
By doing that, Ika provides builders with the flexibility to implement any logic (custody, DeFi, gaming, etc.) across
any blockchain, including Bitcoin.
Builders can now create dApps that have:
```

> * Unrivaled speed: record-breaking throughput and sub-second latency for cross-chain transactions, supporting real-time applications and institutional-scale demand.
> * Unlimited Access: Capable of signing transactions for virtually any blockchain by supporting common signing algorithms - currently ECDSA, soon EdDSA and Schnorr.
> * Zero Trust Security: Users can be cryptographically required for a signature to be generated - noncollusive. No other party can generate a signature without the user.
> * Massive Decentralization: The 2PC-MPC protocol enables participation of hundreds, or even thousands, of signers in the signature process.
> * Native Interoperability: No Bridging, No Wrapping. Controlling native assets directly, allowing for logic enforcement across Web3 without derivatives.

Reviewer: ```suggestion
* Unrivaled speed: record-breaking throughput and sub-second latency for cross-chain transactions, supporting real-time
  applications and institutional-scale demand.
* Unlimited Access: Capable of signing transactions for virtually any blockchain by supporting common signing
  algorithms—currently ECDSA, soon EdDSA and Schnorr.
* Zero Trust Security: Users can be cryptographically required for a signature to be generated—noncollusive.
  No other party can generate a signature without the user.
* Massive Decentralization: The 2PC-MPC protocol enables the participation of hundreds, or even thousands, of signers in
  the signature process.
* Native Interoperability: No Bridging, No Wrapping.
  Controlling native assets directly, allowing for logic enforcement across Web3 without derivatives.
```

## Cryptography of dWallets — 2PC-MPC

> [dWallets](core-concepts/dwallets) use the [2PC-MPC protocol](https://github.com/dwallet-labs/2pc-mpc), a two-party ECDSA protocol we designed specifically for dWallets, where the second party is fully emulated by a network of `N` parties.

Reviewer: ```suggestion
[dWallets](core-concepts/dwallets) use the [2PC-MPC protocol](https://github.com/dwallet-labs/2pc-mpc), a two-party
ECDSA protocol we designed specifically for dWallets, where the second party is fully emulated by a network of `N`
parties.
```

> Besides its novel structure, enabling the non-collusive dWallets and the autonomy and flexibility of a permissionless Network of dWallets (Ika), the 2PC-MPC protocol also dramatically improves upon the latest MPC protocols, allowing the dWallet Network to be scalable and massively-decentralized.

Reviewer: ```suggestion
Besides its novel structure, enabling the noncollusive dWallets and the autonomy and flexibility of a permissionless
Network of dWallets (Ika), the 2PC-MPC protocol also dramatically improves upon the latest MPC protocols, allowing the
dWallet Network to be scalable and massively-decentralized.
```

> The 2PC-MPC protocol achieves linear-scaling in communication — `O(n)` — and due to novel aggregation and amortization techniques, an amortized cost per-party that remains constant up to thousands of parties — practically `O(1)` in computation for the network, allowing it to scale and achieve decentralization, whilst being asymptotically `O(1)` for the user: meaning the size of the network doesn't have any impact on the user as its computation and communication is constant.

Reviewer: ```suggestion
The 2PC-MPC protocol achieves linear-scaling in communication — `O(n)` — and due to novel aggregation and amortization
techniques, an amortized cost per-party that remains constant up to thousands of parties — practically `O(1)` in
computation for the network, allowing it to scale and achieve decentralization, whilst being asymptotically `O(1)` for
the user: meaning the size of the network doesn't have any impact on the user as its computation and communication is
constant.
```
